src/features/collaborator-avatars/CollaboratorList.scss

```diff
@@ -0,0 +1,84 @@
+@import '../../styles/variables';
+
+.collaborator-list {
```

Comment:
Should this be scoped under `usm-collaborator-list` as the render() shows?

Reply:
It's probably a good idea to give it more scope. I'll add that.